tenable/io/v3/vm/plugins.py

```diff
@@ -5,7 +5,7 @@
 The following methods allow for interaction into the Tenable.io
 :devportal:`plugins <plugins>` API endpoints.
 
-Methods available on ``tio.plugins``:
+Methods available on ``tio.v3.vm.plugins``:
```

Comment:
Why is this included with this PR? I would keep this PR focused only on Agent Config

Reply:
Ted. Sure, will ensure this. PR related to plugins is merged, so I thought to include this change in this PR itself as this was a small change.